include/noise/protocol/util.h

```diff
@@ -30,6 +30,8 @@
 extern "C" {
 #endif
 
+int noise_init();
```

Comment:
Ah.  I didn't think of that.  Fix the (void) and leave it in the public API for now and add some docs explaining what the user should do if they have/have-not initialised the external libraries themselves.